### PR TITLE
New login

### PR DESCRIPTION
**Pull request by sayed4900**
Description:
I changed the login page by adding a CSS class called centeredContainer and move Section outside the Grid.
Before
before
oldLoginMob

After
after
newloginMob

[Uploading newlogin.webm…]()

### CODE REVIEW

**Comment by sayed4900** [comment]
> Hey @sayed4900 In our codebase, stack,grid and section are used to maintain consistency across the code. Removing them isn't a good idea as they serve a purpose. We may need to find a better way to fix it. And using using the Carbon Grid makes it easier to have a consistency across the project .Custom classes can create more issues as compared to using the Carbon Grid. Thank You



> Hey @sayed4900 In our codebase, stack,grid and section are used to maintain consistency across the code. Removing them isn't a good idea as they serve a purpose. We may need to find a better way to fix it. And using using the Carbon Grid makes it easier to have a consistency across the project .Custom classes can create more issues as compared to using the Carbon Grid. Thank You

Thanks for your rewive I return the grid again and 

> Hey @sayed4900 In our codebase, stack,grid and section are used to maintain consistency across the code. Removing them isn't a good idea as they serve a purpose. We may need to find a better way to fix it. And using using the Carbon Grid makes it easier to have a consistency across the project .Custom classes can create more issues as compared to using the Carbon Grid. Thank You

Thanks for your review, I return the Grid and Section. 
The changes I made that I move the Section outside the Grid.
Please review it again.

**Comment by mozzy11** [comment]
Thanks @sayed4900 , 
Are the tests you added part of solving the issue ??

**Comment by sayed4900** [comment]
> Thanks @sayed4900 , Are the tests you added part of solving the issue ??

Thanks @mozzy11  for reviewing the pull request.
The tests I added were not directly related to solving the build error.
They were aimed at improving login.
I am now trying to solve the error that related to npm run build, any suggestions to solve it? 
Thanks for your time.

**Comment by mozzy11** [comment]
hello @CalebSLane @caseyi , 
Do you think we need to update the Login Page ??

**Comment by sayed4900** [comment]
> hello @CalebSLane @caseyi , Do you think we need to update the Login Page ??

@mozzy11 @CalebSLane @caseyi 
I take a screen video to show the update 
[Uploading newlogin.webm…]()

Isn't this better?

File: pom.xml [hunk 1]
```diff
@@ -1,582 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (mozzy11) [inline pom.xml:2]: Why did you delete the pom file ??

File: frontend/src/components/Login.js [hunk 2]
```diff
@@ -131,117 +131,120 @@ function Login(props) {
...
     <>
-      <div className="loginPageContent">
+      <div className="loginPageContent  centeredContainer">
         {notificationVisible === true ? <AlertDialog /> : ""}
         <Grid fullWidth={true}>
           <Column lg={0} md={0} sm={4}>
             {loginMessage()}
           </Column>
           <Column lg={4} md={4} sm={4}>
-            <Section>
```
Comment (mozzy11) [inline frontend/src/components/Login.js:141]: can you discard the formating chnages and only add in your css changes ??

**Review by mozzy11 — CHANGES_REQUESTED** [review]
Thanks @sayed4900  for this.

Can you discard the format changes and only reamin with your css changes and also restore the pom file

**Comment by sayed4900** [comment]
@mozzy11 I do the changes that you asked me by when I try to do push, problems happen to me and I do forced push.

This is Login Page now 
Screenshot from 2024-03-28 14-01-43

File: pom.xml (hunk 1 above)
Comment (sayed4900) [reply pom.xml]: I deleted it by wrong sorry about this mistake, I have limited experience with Git.

**Comment by mozzy11** [comment]
is that the screen shot for you latest changes ??
I dont see any update to the login

**Comment by sayed4900** [comment]
> is that the screen shot for you latest changes ?? I dont see any update to the login

yes, this the last changes, the only updates that happen that  **OPENELIS  image** become on the center.
imageedit_2_4205509233

**Comment by sayed4900** [comment]
> Can you just use the Grid and Column to center the login form ??

I really tried to do that from the beginning but I couldn't. I will try again.

**Comment by mozzy11** [comment]
Thabks @sayed4900 
I think we can leave the Logig as it is not
